.github/workflows/release.yml

```diff
@@ -0,0 +1,488 @@
+name: "Release Builds"
+on:
+  push:
+    branches:
+      - v[0-9].[0-9]**
+    tags:
+      - v[0-9].[0-9]**
+      - v[0-9].[0-9]**-**
+
+env:
+  MESON_VERSION: '0.56.2'
+  EM_VERSION: '2.0.29'
+  EM_CACHE_FOLDER: 'emsdk'
+  TAISEI_NOPRELOAD: 0
+  TAISEI_PRELOAD_REQUIRED: 1
+
+jobs:
+  linux-release-build-x64:
+    name: "Linux (x64/Source)"
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Install Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+
+    - name: Install Tools
+      run: >
+        sudo apt update || true # debian cache is not atomic and can sometimes fail
+
+        sudo apt install -y -qq
+        build-essential
+        libsdl2-dev
+        libx11-dev
+        libwayland-dev
+        python3-docutils
+        gnupg
+
+        pip install
+        meson==${{ env.MESON_VERSION }}
+        ninja
+        zstandard
+        python-gnupg
+
+    # NOTE: confusingly, 'fetch-depth: 0' == "get all history", not "no history"
```

Comment:
It's the same as the `--depth` argument to `git clone`. 1 is "one commit of history" (the latest); "no history" wouldn't make sense.

Reply:
Right, but that wasn't intuitive to me from the name of the option, hence the comment.